src/renderers/webgl/WebGLBackground.js

```diff
@@ -56,7 +62,7 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 
 		}
 
-		if ( background && ( background.isCubeTexture || background.isWebGLCubeRenderTarget || background.mapping === CubeUVReflectionMapping ) ) {
+		if ( background && ( background.isCubeTexture || background.isWebGLCubeRenderTargetTexture || background.mapping === CubeUVReflectionMapping ) ) {
```

Comment:
This line here is problematic since its valid to pass an instance of `WebGLCubeRenderTarget` to `Scene.background`. Meaning this code should work: https://jsfiddle.net/73Lcyqf8/1/

Reply:
Ahh, I see..

I don't think passing `WebGLCubeRenderTarget` should be allowed, like in `envMap`.

Will add a warning. Thanks!